### DISCUSSION TITLE
Support for Slack Unfurls when slack integration is set up?

### DISCUSSION THREAD

**Discussion by vaishnavsm** [outline/outline#2676] [Ideas]
Does this exist? Can it exist? Should it exist?

**Comment by tommoor** [comment]
Yes it already exists when the integration is set up correctly

**Reply by vaishnavsm** [reply]
huh ok. I'll have to check my setup. Thanks!

**Comment by aariacarterweir** [comment]
Are you guys sure this works? It's not working for me, I have fully integrated slack (but use Google for SSO)

**Comment by aariacarterweir** [comment]
Hi gang, any update? This is definitely not working and i have the integration all set up properly
